dataplane/vppagent/pkg/vppagent/vppagent.go

```diff
@@ -323,11 +325,38 @@ func (v *VPPAgent) Init(common *common.DataplaneConfigBase, monitor *crossconnec
 	v.setDataplaneConfigVPPAgent(monitor)
 	v.reset()
 	v.programMgmtInterface()
-	v.metricsCollector = metrics.NewMetricsCollector()
-	v.metricsCollector.CollectAsync(monitor, v.vppAgentEndpoint)
+	v.setupMetricsCollector(monitor)
 	return nil
 }
 
+func (v *VPPAgent) setupMetricsCollector(monitor *crossconnect_monitor.CrossConnectMonitor) {
+	val, ok := os.LookupEnv(DataplaneMetricsCollectorEnabled)
+	if !ok {
+		logrus.Infof("%v not set, metrics collector is not using", DataplaneMetricsCollectorEnabled)
+		return
+	}
+	metricsCollectorEnabled, err := strconv.ParseBool(val)
```

Comment:
By default it should be enabled, and please disable by default options in SetupNodes in our tests.

Reply:
thanks, fixed